Block/Adminhtml/Grid/Renderer/Image.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+/**
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the MIT License
+ * It is available through the world-wide-web at this URL:
+ * https://tldrlegal.com/license/mit-license
+ * If you are unable to obtain it through the world-wide-web, please send an email
+ * to support@buckaroo.nl so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade this module to newer
+ * versions in the future. If you wish to customize this module for your
+ * needs please contact support@buckaroo.nl for more information.
+ *
+ * @copyright Copyright (c) Buckaroo B.V.
+ * @license   https://tldrlegal.com/license/mit-license
+ */
+
+namespace Buckaroo\Magento2\Block\Adminhtml\Grid\Renderer;
+
+use Magento\Backend\Block\Widget\Grid\Column\Renderer\AbstractRenderer;
+use Magento\Framework\DataObject;
+use Magento\Store\Model\StoreManagerInterface;
+
+class Image extends AbstractRenderer
+{
+    private $storeManager;
+
+    public function __construct(\Magento\Backend\Block\Context $context, StoreManagerInterface $storemanager, array $data = [])
+    {
+        $this->storeManager = $storemanager;
+        parent::__construct($context, $data);
+        $this->_authorization = $context->getAuthorization();
+    }
+
+    public function render(DataObject $row)
+    {
+        if ($img = $row['logo']) {
+            $mediaDirectory = $this->storeManager->getStore()->getBaseUrl(
+                \Magento\Framework\UrlInterface::URL_TYPE_MEDIA);
+            return '<img src="' . $mediaDirectory . $img . '" width="50"/>';
```

Comment:
no trailing slash is needed in html5, especially without space between double quote

Reply:
fixed